modules/microshift-greenboot-create-health-check-script.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-create-health-check-script_{context}"]
+= Creating a health check script
```

Comment:
This procedure does not really help explain how to create a health check script for their application. It shows how to use the one we have for busybox.

Do we have a separate location for in-depth developer tutorials? Or should we expand this to give more details? If we need to expand it here, maybe we should leave this file out for now and use a separate ticket to write up that procedure.

Reply:
My point is this section is called "Creating a health check script" but the procedure is more like "Downloading a health check script". If we want to write a tutorial for creating an application-specific script, we can do that by including the content of the script in the documentation and explaining how it works. I'm not sure telling them to download the script without reviewing it provides much value.

Reply:
@dhellmann and @ggiguash Do we have "how to use the library of functions Giga created" in the dev docs already, or is this something we need to create? We can keep chipping aways on this PR until about 4/6, then we should finalize and put in review queues. I have the bandwidth, but we should probably meet again.

Reply:
Those instructions do not exist, as far as I know. Maybe we should remove that section for now and focus on the other content in this PR?

Reply:
Ticket created in the 4.14 docs epic.